assets/_scss/pages/_index.scss

```diff
@@ -27,9 +29,103 @@ body {
   }
 }
 
+.perspectives {
+  position: absolute;
+  right: 1em;
+  top: 1em;
+
+  li {
+    @include transition(all 0.1s ease);
+    $toggle-border-color: lighten($color-gray-light, 10);
+    $toggle-thickness: 1px;
+    color: darken($color-gray-light, 10);
+    cursor: pointer;
+    font-size: 0.65em;
+    font-weight: bold;
+    padding: 0.85em 1em;
+
+    &:after {
+      margin-bottom: 0;
+    }
+
+    &:before {
+      content: "";
+      padding-right: 0;
+    }
+
+    &:first-child {
+      border-radius: 0.5em 0 0 0.5em;
+      border-bottom: $toggle-thickness solid $toggle-border-color;
+      border-left: $toggle-thickness solid $toggle-border-color;
+      border-top: $toggle-thickness solid $toggle-border-color;
+    }
+
+    &:hover {
+      color: $color-primary;
+    }
+
+    &:last-child {
+      border-radius: 0 0.5em 0.5em 0;
```

Comment:
Properties should be ordered border-bottom, border-radius, border-right, border-top